README.md

```diff
@@ -1,32 +1,50 @@
 # Bino
 
 The booking bot at your service
+### Requirements
 
-### Configure Database 
+- [Go](https://golang.org/) 1.16
+- [Docker](https://www.docker.com/)
 
-Copy file `.env-sample` to `.env.docker`
+### Configure Database
+
+Both `MYSQL_ROOT_PASSWORD` and `MYSQL_DATABASE` are required to run the database. They are made available to the database via a file named `.env.docker`.
+
+For a quick setup you can copy `.env-sample`:
 
 ```shell
-$ cp .env-sample .env.docker
+cp .env-sample .env.docker
 ```
+### Run the bino bot
 
-Run the command for import dump in `mysql`
+#### Setup Discord Token
+
+Bino needs to connect to a Discord server to run, so you'll need to provide a bot token. Check [WriteBots](https://writebots.com) great [tutorial](https://www.writebots.com/discord-bot-token/) to setup one.
+
+The token needs to be available via an environment variable named `BINO_DISCORD_TOKEN`. You can do it by running:
 
 ```shell
-$ docker exec -i binodb mysql -uroot -pdocker bino < ./db/Bino.sql
```

Comment:
This command we needed, cause, import `.SQL` file in docker container

Reply:
Mysql containers are configured so all files on `docker-entrypoint-initdb.d` folder are executed on startup. So by mount `db` to that same location the server on the containers will execute `Bino.sql` automatically:

![image](https://user-images.githubusercontent.com/2341840/126789760-f0000d69-9ddf-46dd-9b71-c2744d0cd085.png)

This way you don't need to run it manually anymore, I've removed the command.

Do you think we should keep it?